site/specs-temp/numbers.json

```diff
@@ -29980,24 +29992,29 @@
                       "type": "string"
                     },
                     "ActualFocDate": {
+                      "description": "This is the FOC (Final Order Commit) Date assigned to the Port-in Order by the losing carrier or their agent.",
                       "type": "string"
                     },
                     "BillingTelephoneNumber": {
+                      "description": "The BillingTelephoneNumber is the primary telephone number associated with the invoice that the subscriber gets from the losing carrier.",
                       "type": "string"
                     },
                     "CreatedByUser": {
                       "type": "string"
                     },
                     "OrderId": {
+                      "description": "The identifier that uniquely identifies a LSR order.",
```

Comment:
The descriptions don't all use LSR or Lsr. Could we adopt one version for all the descriptions and change the others to match for consistency?